main/src/main/scala/sbt/Defaults.scala

```diff
@@ -2711,6 +2699,18 @@ object Defaults extends BuildCommon {
     }
   )
 
+  private inline def analysisStore: AnalysisStore = {
```

Comment:
An inlined task?

Reply:
Yay that allow me to refactor code in tasks without creating intermediate task. I can use `.value` even if I'm not in a `Def.task`.